src/isomorphic/classic/class/__tests__/ReactClass-test.js

```diff
@@ -355,7 +343,7 @@ describe('ReactClass-spec', function() {
     });
 
     expect(() => Component()).toThrow();
-    expect(console.error.calls.length).toBe(1);
+    expect(console.error.argsForCall.length).toBe(1);
```

Comment:
Yeah, I just noticed the rest of the checks in this file used `argsForCall` so I changed it for consistency. Should I change it back?